benchmarks/run.sh

```diff
@@ -2,22 +2,49 @@
 
 set -xe
 
-WORKSPACE_ROOT="$( cd "$(dirname "$0")/.." ; pwd -P )"
-cd "$WORKSPACE_ROOT"
+function build_benchmarks () {
+  function build_benchmark () {
+    if [ $1 == true ]; then OUTPUT_DIR="dist/size/$2/main"; else OUTPUT_DIR="dist/size/$2"; fi
+    PLATFORM=$2 npm --prefix ./benchmarks run size:build -- -o ${WORKSPACE_ROOT}/benchmarks/$OUTPUT_DIR
```

Comment:
Quoting to avoid issues, e.g. with spaces.

```suggestion
    PLATFORM=$2 npm --prefix ./benchmarks run size:build -- -o "${WORKSPACE_ROOT}/benchmarks/$OUTPUT_DIR"
```